features/css-shapes.md

```diff
@@ -0,0 +1,15 @@
+---
+title: CSS Shapes
+category: css, layout
+bugzilla: 1040714
+firefox_status: in-development
+mdn_url:
+spec_url: http://www.w3.org/TR/css-shapes-1/
+spec_repo:
```

Comment:
Can you add the spec repo for both features? (or remove the `spec_repo` field altogether)